middlewares/validatePostQuestion.js

```diff
@@ -1,19 +1,13 @@
 const validatePostQuestion = (req, res, next) => {
```

Comment:
Add a jSDoc documentation for this method

Reply:
done